libraries/AP_HAL_ChibiOS/hwdef/Aocoda-RC-H743Dual/defaults.parm

```diff
@@ -0,0 +1,4 @@
+# logging
+LOG_BACKEND_TYPE    4
+LOG_FILE_BUFSIZE    4

```

Comment:
that's way too small

Reply:
I found most of `LOG_FILE_BUFSIZE` is set to  16. So how about 16?

```
LOG_FILE_BUFSIZE    16
```

Reply:
Yeah I think remove defaults.parm - the defaults should be set appropriately based on chip (200 for H743 I believe)

Reply:
OK, Thanks a lot.  It seems the param is quite advanced. Just default will do.

https://github.com/ArduPilot/ardupilot/blob/1bf7c9ee776797e160e75295da430b33977db9df/libraries/AP_Logger/AP_Logger.cpp#L26-L36

https://github.com/ArduPilot/ardupilot/blob/1bf7c9ee776797e160e75295da430b33977db9df/libraries/AP_HAL/AP_HAL_Boards.h#L107-L116

https://github.com/ArduPilot/ardupilot/blob/1bf7c9ee776797e160e75295da430b33977db9df/libraries/AP_HAL/board/chibios.h#L7-L19

https://github.com/ArduPilot/ardupilot/blob/1bf7c9ee776797e160e75295da430b33977db9df/libraries/AP_HAL_ChibiOS/hwdef/scripts/chibios_hwdef.py#L1101-L1113

Reply:
now you will need the defaults.parm file to setup the LED, now that it can work and you have it marked as such on new board

Reply:
Welp, I don't know how to, any links or idea about setting LED in defaults.parm file?

Reply:
# setup for LEDs on chan9
SERVO13_FUNCTION 120

Reply:
OK, SERCO13 for PWM(13), thanks very much!